scripts/Muon/GUI/Common/plot_widget/plotting_canvas/plotting_canvas_view.py

```diff
@@ -85,6 +87,9 @@ def create_new_plot_canvas(self, num_axes):
         """Creates a new blank plotting canvas"""
         self._plot_information_list = []
         self._number_of_axes = num_axes
+        self._color_queue = [ColorQueue()]
```

Comment:
I think we may end up with one more ColorQueue object than necessary here, as we initialize the list with one object and then loop over and append num_axes more. Maybe replacing the loop with something like 
```suggestion
        self._color_queue = [ColorQueue() for _ in range(num_axes) ]
```
would suffice.